src/ExpressiveInstaller/config.php

```diff
@@ -167,10 +200,22 @@
                     ],
                     'copy-files' => [
                         '/Resources/config/templates-zend-view.php' => '/config/autoload/templates.global.php',
-                        '/Resources/templates/zend-view-404.phtml' => '/src/App/templates/error/404.phtml',
-                        '/Resources/templates/zend-view-error.phtml' => '/src/App/templates/error/error.phtml',
-                        '/Resources/templates/zend-view-layout.phtml' => '/src/App/templates/layout/default.phtml',
-                        '/Resources/templates/zend-view-home-page.phtml' => '/src/App/templates/app/home-page.phtml',
+                        '/Resources/templates/zend-view-404.phtml' => [
+                            'flat'    => '/templates/error/404.phtml',
+                            'modular' => '/src/App/templates/error/404.phtml',
+                        ],
+                        '/Resources/templates/zend-view-error.phtml' => [
+                            'flat'    => '/templates/error/error.phtml',
+                            'modular' => '/src/App/templates/error/error.phtml',
+                        ],
+                        '/Resources/templates/zend-view-layout.phtml' => [
+                            'flat'    => '/templates/layout/default.phtml',
+                            'modular' => '/src/App/templates/layout/default.phtml',
+                        ],
+                        '/Resources/templates/zend-view-home-page.phtml' => [
+                            'flat'    => '/templates/app/home-page.phtml',
+                            'modular' => '/src/App/templates/app/home-page.phtml',
+                        ],
                     ],
                     'minimal-files' => [
                         '/Resources/config/templates-zend-view.php' => '/config/autoload/templates.global.php',
```

Comment:
Can we remove this `minimal-files` options and only have the `copy-files` key?

                    'copy-files' => [
                        '/Resources/config/templates-zend-view.php' => '/config/autoload/templates.global.php',
                        '/Resources/templates/zend-view-404.phtml' => [
                            'flat'    => '/templates/error/404.phtml',
                            'modular' => '/src/App/templates/error/404.phtml',
                        ],
                        '/Resources/templates/zend-view-error.phtml' => [
                            'flat'    => '/templates/error/error.phtml',
                            'minimal' => '/src/App/templates/error/error.phtml',
                        ],
                    ],

If the source key is a single string it's always copied. If it's an array if will only be copied for `flat`, `modular`, `minimal`. This makes the config a bit easier and I'm thinking about a possible future disco integration here.